client/package.json

```diff
@@ -20,11 +18,23 @@
   },
   "homepage": "https://github.com/jennie-quinton/dndmaster#readme",
   "dependencies": {
-    "@zeit/next-sass": "^1.0.1",
-    "next": "^8.0.3",
+    "css-loader": "^2.1.1",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.5.0",
     "node-sass": "^4.11.0",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.29.5",
+    "webpack-dev-server": "^3.2.1"
```

Comment:
Move this to `devDependencies` please. The reason is due to production server does not need to install this, therefore making the install command slightly quicker.